README.md

```diff
@@ -1 +1,54 @@
-# microverse
\ No newline at end of file
+# Hello-Microverse
+
+## "Hello Microverse" project
+
+the purpose of this project is for mastering best practise, familiar with all the tools used such as linters, github, html and css
+
+### Built With
+
+- HTML and CSS
+
+#### technology used
+
+1. Linters
+2. VScode
+3. html & css
+4. github
+
+### Getting Started
+
+1. copy the Url from my github account
+2. clone it to your local machine using
+
+### Prerequisites
+
+1. Setup
+2. Install
+3. Usage
+4. Tun tests
+5. Deployment
+
+#### Live Demo
+
+Link:<http://127.0.0.1:5500/index.html>
```

Comment:
- [ ] In `.md` files, you should use `[link name](actual link)`, you can read more about it [here](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

Reply:
This should just be a note for improvement.